tests/Feature/HostedDomainsTest.php

```
@@ -523,4 +525,117 @@ public function test_matching_ssls_returns_empty_for_invalid_domain(): void
+
+        $this->post(route('hosted-domains.check-expiry-all', [
+            'server' => $this->server->id,
+            'site' => $this->site->id,
+        ]))
+            ->assertRedirect()
+            ->assertSessionHas('success');
+
+        $ssl->refresh();
+        $this->assertTrue($ssl->expires_at->isAfter(now()->addDays(80)));
+        Notifier::shouldNotHaveReceived('send');
+    }
```

PHPMD (2.15.0)

[error] The method test_check_expiry_all_refreshes_every_site_certificate is not named in camelCase. (undefined)

(CamelCaseMethodName)